src/plot_utils.py

```diff
@@ -88,6 +88,18 @@ def heatmap(bdb, bql, vmin=None, vmax=None, row_ordering=None,
         Active BayesDB instance.
     bql : str
         The BQL to run and plot. Must be a PAIRWISE BQL query.
+    **kwargs : dict
+        Passed to df_heatmap
+    Returns
+    -------
+    clustermap: seaborn.clustermap
+    """
+    df = bqlu.cursor_to_df(bdb.execute(bql))
+    df.fillna(0, inplace=True)
+    return df_heatmap(df, **kwargs)
+
+def df_heatmap(df, vmin=None, vmax=None, row_ordering=None, col_ordering=None):
```

Comment:
This function belongs in the section commented as [`internal`](https://github.com/probcomp/bdbcontrib/blob/20150929-1446-gremio-quickstart/src/plot_utils.py#L229), since it is not exposed in the API.